src/index.js

```diff
@@ -15,6 +15,12 @@ class IframeResizer extends React.Component {
     this.updateIframe(this.props);
     this.resizeIframe(this.props);
   }
+  componentWillUnmount() {
+    // React will remove the iframe, however we need to manually
+    // call iframe-resizer to stop its listeners
+    iframeResizer = this.refs.frame.iFrameResizer
```

Comment:
@zeroasterisk this line is erroring for me 

`iframeResizer is not defined`

It needs to be changed to:

`let iframeResizer = this.refs.frame.iFrameResizer`

Reply:
Updated PR #13 to use `const`